api/cmd/html/index.html

```diff
@@ -0,0 +1,36 @@
+<!--
```

Comment:
we should remove this file. an build it form source when building a binary.

Reply:
same as https://github.com/apache/apisix-dashboard/pull/1788#discussion_r634528665

As we have to provide an option not to embed the web build. We can't leave an empty directory.

Reply:
not to embed empty dir, build `html` before embed it

Reply:
hi, are you sure? Won't that be time-consuming during testing and development?
BTW, the file gets replaced by the actual build when we issue a full build.
let me know what you think.

Reply:
Sure. Maybe we could add a parameter, `make api-run` does not need to embed, `make build` does.

Reply:
Yes, that's exactly what I have added. 
`make build` full build with the web embed.
`make api-run` run without embed.
[new] `make api-build` build(just the manager-api) without embed.

Now as you see, we have 2 cases when we are not building web assets. maybe as a substitute for this current `index.html`, we can create an empty directory `html` and an empty file, maybe `keep.keep`. This would work fine. But I thought, instead of an empty file an HTML with a suitable message (`The build is partial`) would be appropriate.
So should we go with the other option just I mentioned?

Reply:
Just a thought, How about adding the dag-to-lua package as a git submodule. instead of downloading and unzipping.

Reply:
> Yes, that's exactly what I have added.

so why do you think that will be time-consuming?

We cannot embed old files, but files built from the current source.

Reply:
Oh, actually I thought for every build, you wish to build HTML
> We cannot embed old files, but files built from the current source.

Yes the current code also ensures that too[ref](https://github.com/apache/apisix-dashboard/pull/1788/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R46-R49)
`	rm -rf api/cmd/html && git checkout api/cmd/html/ && api/build.sh && mkdir -p output/logs`

The `rm -rf` and `checkout` is helpful because I have excluded all build files except `index.html` from the HTML dir.

To be honest, the static index.html serves two purpose
1. preventing the error from go embed the empty directory.
2. Instead of filling the directory with some garbage file, an HTML with a message seems reasonable. As anyways it will get replaced by web build's index.html.

Let me know what changes should I make